pkg/kudoctl/util/repo/repo_framework.go

```diff
@@ -67,13 +60,13 @@ func (r *FrameworkRepository) DownloadIndexFile() (*IndexFile, error) {
 }
 
 // GetPackage downloads the tgz file from the given repo
-func (r *FrameworkRepository) GetPackage(bundleName string) (*PackageCRDs, error) {
+func (r *FrameworkRepository) GetPackage(packageName string) (*PackageCRDs, error) {
```

Comment:
to my comment from above: 
```suggestion
func (r *FrameworkRepository) DownloadPackageFile(packageName string) (*PackageCRDs, error) {
``` ?

Reply:
Again, it downloads, extracts, unmarshals and fills the `PackageCRDs` struct representing the framework. Though more generic, `GetPackage` captures this more accurately.